packages/shared/lib/core/wallet/actions/getAccountAssetsForSelectedAccount.ts

```diff
@@ -103,28 +101,7 @@ function getAccountAssetForChain(chainId: number): IAccountAssetsPerNetwork | un
                     ...persistedAsset,
                     balance: _balance,
                     chainId,
-                    standard: 'Layer 2 Native Token',
```

Comment:
We should also update the check above here 
```
if (persistedAsset && persistedAsset?.metadata && isValidIrc30(persistedAsset.metadata))
```
What makes an erc20 token valid? The ethereum documentation says

> Keep in mind that there are no floats in Solidity. Therefore most tokens adopt 18 decimals and will return the total supply and other results as followed 1000000000000000000 for 1 token

but the decimals could theoretically be up to 255 since its a `uint8`  https://eips.ethereum.org/EIPS/eip-20

Reply:
Althought theoretically it could be more than 18, I think it's sensible to cap it to 18 here. There is no reason to allow a lot more, since most tokens just copy the default value (18). Additionally, on the Shimmer EVM, only the last 6 decimals are used.

See `Danger` part: https://wiki.iota.org/shimmer/smart-contracts/guide/evm/compatibility/

Reply:
gotcha, thanks!